packages/airplane/internal/config.ts

```diff
@@ -520,3 +530,146 @@ const normalizeConfig = <TParams extends InputParams>(
     name: config.name ? config.name : humanizeString(config.slug),
   };
 };
+
+/**
+ * The configuration for a SQL Task. To learn more, see the
+ * [Task definition reference](https://docs.airplane.dev/tasks/task-definition).
+ *
+ * Builtin task bindings are currently in beta.
```

Comment:
```suggestion
 * Declaring SQL tasks with JavaScript is currently in beta.
```

Unrelated, but this made me notice `Airplane Views are currently in beta.` is outdated -- can you remove that?